CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 * Verbose info is missing for partial overlay ID [moby/moby#35989](https://github.com/moby/moby/pull/35989)
 * Update `FindNetwork` to address network name duplications [moby/moby#30897](https://github.com/moby/moby/pull/30897)
 * Disallow attaching ingress network [docker/swarmkit#2523](https://github.com/docker/swarmkit/pull/2523)
+- Fix automatic removal of ingress sandbox when last service leaves [moby/moby#36538](https://github.com/moby/moby/pull/36538)
```

Comment:
This Change has a description for changelog provided in the moby/moby pullrequest
That's: 
```
Prevent implicit removal of the ingress network
```

Reply:
updated